### ISSUE TITLE
flags not working with color_edit4

### ISSUE THREAD

**Issue by djblackops** [pyimgui/pyimgui#256]
hey there, i've just setup pyimgui with the pyglet backend on my program.

im using `imgui.color_edit4` like this:

```
imgui.begin("test window")
imgui.text("test")
changed, color = imgui.color_edit4("Test", 1.0, 1.0, 1.0, 1.0, False, imgui.COLOR_EDIT_NO_INPUTS)
imgui.end()
```

the flags just seem to be ignored and im still getting the color sliders like so:
![python_MgdMNF7uQS](https://user-images.githubusercontent.com/48581418/152020354-664c5450-8f17-4d87-bc74-07677f5a9f7e.png)

im using master branch and tried with the dev branch as well, but im still having the same issue.

**Comment by KinoxKlark** [comment]
Indeed it is clearly a bug, thanks!

Passing `False` as `show_alpha` argument was disabling all flags due to bad implementation of retro compatibility feature. This boolean isn't part of the DearImGui interface anymore and can be achieved using flags. So I just removed it so that the only way to call these functions is using flags.

I proposed a fix in #257. I'll wait to see if all tests pass and I'll merge the PR into the `dev/version-2.0` branch.
